index.html

```diff
@@ -8,21 +8,30 @@
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"/>
 </head>
-<body>
-    <header class="navigation">
-        <nav class="header">
-            <a class="logo" href="#">My Logo</a>
-            <a href="#" class="menu"><img src="img/menu.png" alt=""></a>
-        </nav>
-    </header>
+<body> 
```

Comment:
[OPTIONAL] You have done an amazing job with this project 👍 . I think it would be a good idea to use a more descriptive name for your feature branch. For example, instead of using "works-branch", you can call it "add-works-section", "finish-mobile-version", etc. Always keep this in mind for future projects. This will help make your PR look more professional 👌.